.gitignore

```diff
@@ -86,6 +86,7 @@ celerybeat-schedule
 .venv
 venv/
```

Comment:
Please add this file in a PR in which you will add .gitignore. 

Always try to create a PR to solve the specific issues. All the other files should be added with different issues pointing to that PR. :)

Reply:
Okay! :D